tests/Functional/Doctrine/Mock/DriverConnectionMock.php

```
@@ -20,47 +20,43 @@
     {
         return new StatementMock();
     }
 
-    public function query(?string $sql = null): Result
+    public function query(string $sql): Result
     {
         return new DriverResultMock();
     }
 
-    public function quote($input, $type = ParameterType::STRING)
+    public function quote($value, $type = ParameterType::STRING)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$type'. (undefined)

(UnusedFormalParameter)